spring-cloud-gcp-starters/spring-cloud-gcp-starter-logging/src/main/java/org/springframework/cloud/gcp/logging/autoconfig/TraceIdExtractorType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.logging.autoconfig;
+
+/**
+ * Enum values represent the order and combination of predefined trace ID extractors that
+ * can be used.
+ *
+ * @see XCloudTraceIdExtractor
```

Comment:
All of these class references are invalid because they're not fully qualified and are not imported in this file.
Use something like `@see org.springframework.cloud.gcp.logging.XCloudTraceIdExtractor`.